wp-content/plugins/tvarchive-politicalads/tvarchive-politicalads.php

```diff
@@ -605,13 +605,13 @@ function get_candidates() {
     // TODO: Cache the results of this query
     global $wpdb;
     $table_name = $wpdb->prefix . 'postmeta';
-    $query = "SELECT count(*) as ad_count,
-                     meta_value as ad_candidate
-                FROM ".$table_name."
-               WHERE meta_key LIKE 'ad_candidates_%_ad_candidate'
-                 AND post_id IN (select ID from wp_posts where post_status = 'publish')
-            GROUP BY meta_value
-            ORDER BY ad_count desc";
+    $query = "SELECT count(*) as ad_count, a.meta_value as ad_candidate, b.meta_value as ad_race from ".$table_name." a
```

Comment:
This is nit picky but it's good practice to separate each column you are selecting to a separate line, and also to have the "FROM" clause be on its own line.  It just increases legibility of the query and prevents overly long lines.